report_xlsx_helper/__manifest__.py

```diff
@@ -7,7 +7,7 @@
               'Odoo Community Association (OCA)',
     'website': 'https://github.com/OCA/reporting-engine',
     'category': 'Reporting',
-    'version': '12.0.1.1.1',
+    'version': '12.0.1.1.2',
```

Comment:
@Saran440 AFAIK, I think you don't need to increase the version. ocabot will do it auto when PSC merge it.

Reply:
Fixed it. Thank you for your reviewed.